app/livechat/imports/server/rest/sms.js

```diff
@@ -1,11 +1,26 @@
+import { HTTP } from 'meteor/http';
 import { Meteor } from 'meteor/meteor';
 import { Random } from 'meteor/random';
 
+import { FileUpload } from '../../../../file-upload';
 import { LivechatRooms, LivechatVisitors, LivechatDepartment } from '../../../../models';
 import { API } from '../../../../api/server';
 import { SMS } from '../../../../sms';
 import { Livechat } from '../../../server/lib/Livechat';
 
+const fileStore = FileUpload.getStore('Uploads');
+
+const getUploadFile = async (details, fileUrl) => {
+	try {
+		const response = HTTP.get(fileUrl, { npmRequestOptions: { encoding: null } });
+		const buffer = new Buffer(response.content);
```

Comment:
you don't need to create a new `Buffer` because `response.content` is already a `Buffer`.

but in case you need to create a new `Buffer` from some content, please use `Buffer.from(content)` instead, as `new Buffer(content)` is deprecated, see https://nodejs.org/dist/latest-v12.x/docs/api/buffer.html#buffer_new_buffer_string_encoding

Reply:
Yeah, you're right. I was the `new Buffer` to get the content size, but since the content is already a `Buffer`,  I can rely on the `content.length` prop.

Thanks.